src/util/cable_runtime_opts_mod.F90

```diff
@@ -107,13 +110,15 @@ MODULE cable_runtime_opts_mod
   LOGICAL :: or_evap = .FALSE.
   LOGICAL :: test_new_gw = .FALSE.
   LOGICAL :: sync_nc_file = .FALSE.
-  INTEGER :: max_spins = -1
+  INTEGER :: max_spins = -1 ! MMY-phase2 change from -1 to 30 - rk4417
   LOGICAL :: fix_access_roots = .FALSE.  !use pft dependent roots in ACCESS
   !ACCESS roots
   LOGICAL :: access13roots = .FALSE.     !switch to use ACCESS1.3 %froot
 
   LOGICAL :: l_limit_labile = .FALSE.    ! #237: limit Labile in spinup
   LOGICAL :: NtilesThruMetFile = .FALSE. ! #199: Specify Ntiles thru met file
+  !line below inserted to fix compilation error - rk4417 - phase2
+  INTEGER :: force_npatches_as=-1 
```

Comment:
This new namelist option needs an entry in the documentation to explain what it is.

It does not seem to be related to the GW so I would rather introduce it separately than within the GW changes.

Reply:
All I can say here is that I needed this line to get the code to compile at the time. Perhaps the present code will compile without it, I don't know. I suggest we leave this bit till the end to see whether it is still needed to get the code to compile.